app/controllers/impressions_controller.rb

```diff
@@ -39,6 +39,14 @@ def update
     end
   end
 
+  def ratio_interacted
+    @impression = Impression.find(params[:id])
+    @ad_unit = AdUnit.find(params[:id])
```

Comment:
"placeholder" logic for time being, until we get impressions controller up and running